src/web/extension.ts

```diff
@@ -4,19 +4,66 @@
  * ------------------------------------------------------------------------------------------ */
 
 import * as vscode from "vscode";
+import { dataverseAuthentication } from "./common/AuthenticationProvider";
+import { appTypes, PORTALSURISCHEME, PORTALSWORKSPACENAME } from "./common/Constants";
 
 export function activate(context: vscode.ExtensionContext): void {
-    console.log("Activated web extension!"); // sample code for testing the webExtension
+    console.log("Activated Power Portal vscode web extension!"); // sample code for testing the webExtension
 
     context.subscriptions.push(
         vscode.commands.registerCommand(
             "microsoft-powerapps-portals.webExtension.init",
             () => {
-                // sample code for testing the webExtension
                 vscode.window.showInformationMessage(
-                    "Initializing web extension!"
+                    "Initializing Power Portal vscode web extension!"
                 );
             }
         )
     );
+
+    let initialized = false;
+    let accessToken = '';
+    context.subscriptions.push(vscode.commands.registerCommand('portals.init', async (args: any) => {
```

Comment:
Is this command declared in package.json file? I think the builds are failing due to this. Please run 'gulp ci' in local to verify your changes.

Reply:
Moreover why do we need this method...we can use the 'microsoft-powerapps-portals.webExtension.init' registered above.